### PR TITLE
Add test to cover more of storage.backends.disk.py

### PR DESCRIPTION
**Pull request by haniffm**

### CODE REVIEW

File: ESSArch_Core/storage/backends/tests/test_disk.py [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+        expected_dest = os.path.join(self.datadir, "some/object/path")
+        expected_copy_calls = [mock.call(src, expected_dest, block_size=65536) for src in src_list]
+
+        self.assertEqual(mock_copy.call_count, 3)
+        mock_copy.assert_has_calls(expected_copy_calls)
+        mock_st_obj.assert_called_once()
+
+    @mock.patch("ESSArch_Core.storage.models.StorageObject")
+    @mock.patch("ESSArch_Core.storage.backends.disk.copy")
+    def test_read_(self, mock_copy, mock_storage_obj):
```
Comment (OskarPersson) [inline ESSArch_Core/storage/backends/tests/test_disk.py]: `test_read` or a more descriptive name?

Comment (haniffm) [reply ESSArch_Core/storage/backends/tests/test_disk.py]: good catch, will fix!

File: ESSArch_Core/storage/backends/tests/test_disk.py [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+
+        mock_copy.assert_called_once_with("the_full_path", "some_dest", block_size=65536)
+
+    @mock.patch("ESSArch_Core.storage.models.StorageObject")
+    @mock.patch("ESSArch_Core.ip.models.InformationPackage")
+    @mock.patch("ESSArch_Core.storage.backends.disk.copy")
+    def test_read_container_default(self, mock_copy, mock_ip, mock_storage_obj):
+        disk_storage_backend = DiskStorageBackend()
+        full_path_to_src = "the/full/path/to/src"
+        container_path = os.path.join("container/path")
```
Comment (OskarPersson) [inline ESSArch_Core/storage/backends/tests/test_disk.py]: `os.path.join` has no effect here

```suggestion
        container_path = "container/path"
```

File: ESSArch_Core/storage/backends/tests/test_disk.py [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+        mock_copy.assert_called_once_with(f"{full_path_to_src}", "some_dest", block_size=65536)
+
+    @mock.patch("ESSArch_Core.storage.backends.disk.DiskStorageBackend._extract")
+    @mock.patch("ESSArch_Core.storage.models.StorageObject")
+    @mock.patch("ESSArch_Core.ip.models.InformationPackage")
+    @mock.patch("ESSArch_Core.storage.backends.disk.copy")
+    def test_read_container_extract(self, mock_copy, mock_ip, mock_storage_obj, mock_extract):
+        disk_storage_backend = DiskStorageBackend()
+        full_path_to_src = "the/full/path/to/src"
+        container_path = os.path.join("container/path")
```
Comment (OskarPersson) [inline ESSArch_Core/storage/backends/tests/test_disk.py]: `os.path.join` has no effect here

```suggestion
        container_path = "container/path"
```

**Pull request merged**